app/src/main/java/com/iet/ietians_diary/Models/RoadmapModels.java

```diff
@@ -1,10 +1,15 @@
 package com.iet.ietians_diary.Models;
 
+import android.graphics.Color;
+import android.graphics.drawable.Drawable;
+
 public class RoadmapModels {
```

Comment:
I guess it would be Roadmap (singular) instead of Roadmaps, as it's a model for a single roadmap.

Reply:
Also, please rename the file too @adarsh434

Reply:
I made the changes, thanks for the review.